coresimd/ppsv/api/masks.rs

```diff
@@ -11,25 +11,29 @@ macro_rules! impl_mask_minimal {
             /// Creates a new instance with each vector elements initialized
             /// with the provided values.
             #[inline]
+            #[promotable_const_fn]
             pub const fn new($($elem_name: bool),*) -> Self {
```

Comment:
which stable code can call the const functions in this file?

Reply:
is it possible to find that out?

Reply:
Uh... other than removing the attribute and following the fallout?

Reply:
Could you post the fallout? I am curious because nothing in this module should be reachable from stable code.

Reply:
Actually, only the stdsimd/arch/detect/cache.rs functions should be reachable from stable code =/

Reply:
Oh... will do, I might have been a little overeager in marking things here, because it was hard to tell what was called from where.

Reply:
Might be that I screwed up. Worst case we can just merge this as is, and once the next nightly I can try removing the attribute from all places where it shouldn't be needed. Let me know when you want this merged.

Reply:
ok, removed all unnecessary attributes